backend/program/scrapers/__init__.py

```diff
@@ -55,3 +56,13 @@ def _needs_new_scrape(self, item) -> bool:
             > scrape_time
             or item.scraped_times == 0
         )
+    def _check_for_title_match(self, item, string) -> bool:

```

Comment:
whats this, its not being used?

Reply:
It's going to be, just getting it ready to switch jackett/prowlarr to keyword queries